src/PrestaShopBundle/Service/Hook/HookFinder.php

```diff
@@ -71,21 +71,23 @@ public function find()
         if (!is_array($hookContent)) {
             $hookContent = array();
         }
-        
+
         foreach ($hookContent as $moduleName => $moduleContents) {
-            foreach ($moduleContents as $content) {
-                if (!$content instanceof HookContentClassInterface) {
-                    throw new \Exception('The class returned must implement HookContentClassInterface');
-                }
-                
-                // Check data returned if asked
-                if (!count($this->expectedInstanceClasses)) {
-                    continue;
-                }
-                if (is_object($content) && !in_array(get_class($content), $this->expectedInstanceClasses)) {
-                    throw new \Exception('The module '.$moduleName.' did not return expected class. Was '.get_class($content).' instead of '.implode(' or ', $this->expectedInstanceClasses).'.');
-                } elseif (!is_object($content)) {
-                    throw new \Exception('The module '.$moduleName.' did not return expected type. Was '.gettype($content).' instead of '.implode(' or ', $this->expectedInstanceClasses).'.');
+            if (is_array($moduleContents)) {
```

Comment:
It would be better to have

```php
if (!is_array($moduleContents)) {
    continue;
}
```

to avoid more and more indentation don't you think ?